src/base/bittorrent/session.cpp

```diff
@@ -329,7 +329,13 @@ Session::Session(QObject *parent)
     , m_IPFilterFile(BITTORRENT_SESSION_KEY("IPFilter"))
     , m_announceToAllTrackers(BITTORRENT_SESSION_KEY("AnnounceToAllTrackers"), false)
     , m_announceToAllTiers(BITTORRENT_SESSION_KEY("AnnounceToAllTiers"), true)
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+    , m_asyncIOThreads(BITTORRENT_SESSION_KEY("AsyncIOThreadsCount"), 10)
+    , m_hashingThreads(BITTORRENT_SESSION_KEY("HashingThreadsCount"), 2)
+#else
     , m_asyncIOThreads(BITTORRENT_SESSION_KEY("AsyncIOThreadsCount"), 4)
+    , m_hashingThreads(BITTORRENT_SESSION_KEY("HashingThreadsCount"), -1)
```

Comment:
You should make this line unconditional on libtorrent version. This code isn't good for someone who upgrade qbt (that also upgraded libtorrent 1.2 -> 2.0).